templates/.vscode/launch.json

```diff
@@ -1,32 +1,48 @@
 {
-    // Use IntelliSense to learn about possible attributes.
-    // Hover to view descriptions of existing attributes.
-    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-    "version": "0.2.0",
-    "configurations": [
-        {
-            "type": "node",
-            "request": "attach",
-            "name": "Attach to Remote",
-            "address": "127.0.0.1",
-            "port": 9229,
-            "localRoot": "${workspaceFolder}"
-        },
-        {
-            "name": "Unit Tests",
-            "type": "node",
-            "request": "launch",
-            "protocol": "inspector",
-            "program": "${workspaceRoot}/node_modules/.bin/_mocha",
-            "args": [
-                "--require", "test/helpers/init.js",
-                "--require", "ts-node/register",
-                "--require", "source-map-support/register",
-                "--recursive",
-                "--reporter", "spec",
-                "test/**/*.test.ts"
-            ],
-            "cwd": "${workspaceRoot}"
-        }
-    ]
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "node",
+      "request": "attach",
+      "name": "Attach to Remote",
+      "address": "127.0.0.1",
+      "port": 9229,
+      "localRoot": "${workspaceFolder}",
+      "remoteRoot": "${workspaceFolder}"
```

Comment:
Fixes issue with not being able to connect to the open debug port when running the command in debug mode (e.g. with `--dev-suspend`)
![Screen Shot 2019-07-16 at 8 53 36 AM](https://user-images.githubusercontent.com/1041842/65047073-9e4f0f00-d916-11e9-8dae-0fabb150034f.png)